overlord/overlord_test.go

```diff
@@ -150,7 +150,7 @@ func (ovs *overlordSuite) TestEnsureLoopRunAndStop(c *C) {
 	case <-time.After(2 * time.Second):
 		c.Fatal("Ensure calls not happening")
 	}
-	c.Check(time.Since(t0) >= 20*time.Millisecond, Equals, true)
+	c.Check(time.Since(t0) >= 10*time.Millisecond, Equals, true)
 
```

Comment:
as I said the change in Loop requires more changes in other tests because they don't test what they were testing anymore (they just test the first Ensure)